eventonica-frontend/src/components/Events.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect, useReducer } from 'react';
+import AddEvent from './AddEvent';
+import DeleteEvent from './DeleteEvent';
+
+//Define initialState and reducer function to track fields of newly added events
+const initialState = {
+  id: "",
```

Comment:
Generally in JS you'll always use single quotes everywhere. Another nit from the [Airbnb style guide](https://github.com/airbnb/javascript#strings--quotes).